src/v2/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -181,7 +181,7 @@ const SavedAddresses: React.FC<SavedAddressesProps> = props => {
         modalDetails={modalDetails}
         closeModal={() => setShowAddressModal(false)}
         address={address}
-        onSuccess={() => onSuccess}
+        onSuccess={() => onSuccess()}
```

Comment:
Was this callback not being fired without the () call?

Reply:
Yes that was it 😄  I also moved the `closeModal` after the `onSuccess` call but not sure if that was the problem but kept it there for consistency. 

I also looked at the relay store updater. It looks and runs correctly but it doesn't do anything so a refetch through this `onSuccess` method is needed. I will take another look at it